client/adjudicate.go

```diff
@@ -204,11 +207,8 @@ func (c *Channel) ForceUpdate(ctx context.Context, update func(*channel.State) e
 
 	// Update state
 	state := ar.Tx.State.Clone()
+	updater(state)
```

Comment:
I notice that, unlike the `channel.Update` function, we do not validate the state after applying the `updater`.

Is it not necessary to validate the state, before sending an on-chain transaction ? Because, if the `updater` has modified the state in unexpected ways (say, decrementing version number), then this would mean sending a transaction, that would fail for sure.

Reply:
You are right. Added a quick fix for that.

Reply:
The fix looks good to me.